tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/TelemetryTests.cs

```diff
@@ -68,7 +68,7 @@ public async Task Telemetry_WithAgentProxy_IsSentOnAppClose()
             using var agent = MockTracerAgent.Create(Output, useTelemetry: true);
             Output.WriteLine($"Assigned port {agent.Port} for the agentPort.");
 
-            EnableTelemetry();
+            EnableAgentProxyTelemetry();
```

Comment:
I think we should update this test to test on all the different transports inc named pipe, UDS etc, especially as we're going to be using named pipes in AAS now

Reply:
So, turns out, I already did that! 😅